models/platform-limits/README.md

```diff
@@ -12,6 +12,251 @@ $ pnpm add -D @commercetools-test-data/platform-limits
 
 # Usage
 
+- [BusinessUnitLimitsProjection](#businessunitlimitsprojection)<br>
+- [CartDiscountLimitsProjection](#cartdiscountlimitsprojection)<br>
+- [CartLimitsProjection](#cartlimitsprojection)<br>
+- [CustomerLimitsProjection](#customerlimitsprojection)<br>
+- [CustomerGroupLimitsProjection](#customergrouplimitsprojection)<br>
+- [PlatformLimits](#platformlimits)<br>
+- [LimitWithCurrent](#limitwithcurrent)<br>
+- [ProductDiscountLimitsProjection](#productdiscountlimitsprojection)<br>
+- [ShippingMethodLimitsProjection](#shippingmethodlimitsprojection)<br>
+- [ShoppingListLimitsProjection](#shoppinglistlimitsprojection)<br>
+- [StoreLimitsProjection](#storelimitsprojection)<br><br>
+- [TaxCategoryLimitsProjection](#taxcategorylimitsprojection)<br>
+- [ZoneLimitsProjection](#zonelimitsprojection)
+
+## `BusinessUnitLimitsProjection`
+
+```ts
+import {
+  BusinessUnitLimitsProjection,
+  type TBusinessUnitLimitsProjection,
+} from '@commercetools-test-data/product-type';
```

Comment:
It seems the import is using a wrong package.

Reply:
Done https://github.com/commercetools/test-data/pull/657/commits/6bfab6455c8b504b50e7e12d8b8c3e1da173dcef#diff-c5cb7d76dd373d32885396efb72c25d87c7d548a6671dba26273537a306cdfa4